website/docs/reference/widgets/table/README.md

```diff
@@ -60,183 +114,231 @@ Now, the table widget should be populated with the data coming from the query.
-| **tableData**          | This property gets all the table data in JSON format.                                                                        | `{{<table_name>.tableData}}`           |
-| **selectedRowIndex**   | It gives the index of the row selected by the user. Not applicable when multiple rows are selected.                          | `{{<table_name>.selectedRowIndex}}`    |
-| **selectedRowIndices** | It gives you an array of the index of the rows selected by the user. Not applicable when multi-row selection is turned off.    | `{{<table_name>.selectedRowIndices`\}} |
-| **filteredTableData**  | Contains the data of the rows left after applying any selected filters, sort rule, or search terms.                  | `{{<table_name>.filteredTableData}}`   |
-| **pageNo**             | Contains the current page number that the user is on. APIs can use it for pagination                                         | `{{<table_name>.pageNo}}`              |
-| **pageSize**           | Contains the number of rows that can fit inside a page of the table. Changes along with the height & row height of the table | `{{<table_name>.pageSize}}`            |
-| **searchText**         | Contains the search text entered by the user in the Table                                                                    | `{{<table_name>.searchText}}`          |
+The **pageOffset** property (as well as **pageNo** and **pageSize**) can be used in the API / query by referencing it inside curly braces `{{ }}`.
 
-For the following examples of binding properties, consider a table called `Table_1` with the table data given below:
+```sql
+SELECT * FROM users LIMIT {{ Table1.pageSize }} OFFSET {{ Table1.pageOffset }}
+```
 
 ```
-[
-  {
-    "step": "#1",
-    "task": "Drop a table",
-    "status": "✅",
-    "action": ""
-  },
-  {
-    "step": "#2",
-    "task": "Create a query fetch_users with the Mock DB",
-    "status": "--",
-    "action": ""
-  },
-  {
-    "step": "#3",
-    "task": "Bind the query using => fetch_users.data",
-    "status": "--",
-    "action": ""
-  }
-]
+https://mock-api.appsmith.com/users?page={{Table1.pageNo}}
 ```
 
-And this is how the data looks in the table widget:
+### Key based pagination
 
-![](/img/table\_sample\_data.png)
+This method uses a value in the response of the API as the key to the next API call. This can be configured in the API settings by providing the Next & Previous URLs that the API should execute **onPageChange**.
 
-**selectedRow**
+![](</img/pagination_(2)_(2).gif>)
 
-A table you have created might contain a large amount of data. But when you want to get the data of a single row of the table, the selectedRow property comes into play.\
-selectedRow is an internal property that fetches the data of the table row that the user selected. You can bind different widgets to each table row using this property.
+## Server side searching
```

Comment:
the content in this section appears a bit verbose. need to condense

Reply:
Updated, how's this?

> Server side searching is useful for reducing unnecessary results from queries: rather than requesting lots of data from the server and then filtering it on the client, you can pass search terms to the server so it only fetches relevant results in the first place. This can significantly improve response times when working with large data sets.